pages/api/episode/[id].ts

```diff
@@ -0,0 +1,20 @@
+import { NextApiRequest, NextApiResponse } from "next";
+import {
+  createPodcastEntry,
+  getPodcastEntry,
+  withDatabase,
+} from "../../../server/db";
+
+export default withDatabase(
+  async (req: NextApiRequest, res: NextApiResponse) => {
+    const { id } = req.query;
+    const result = await getPodcastEntry(id);
+    if (!result) {
+      const newPodcastEntry = await createPodcastEntry(id);
+      console.log("new");
```

Comment:
```suggestion
      console.log("new");
```

do you need this anymore?